docs/plugin.md

```diff
@@ -0,0 +1,15 @@
+# Plugin
+
+```{eval-rst}
+.. currentmodule:: scipy_doctest.plugin
+.. autosummary::
+    :nosignatures:
+    :toctree: generated
+
+    DTModule
+    DTTextfile
+    pytest_addoption
+    pytest_configure
+    pytest_ignore_collect
+    pytest_collection_modifyitems
```

Comment:
Same comment: I'm not sure it's useful to dump docstrings of these functions/classes to users. We probably should add some free-form explanations of how we do things, possibly in a sequel. But these functions, I don't think.